MdePkg/Include/Library/DebugLib.h

```diff
@@ -8,6 +8,13 @@
   of size reduction when compiler optimization is disabled. If MDEPKG_NDEBUG is
   defined, then debug and assert related macros wrapped by it are the NULL implementations.
 
+  The NULL implementations of these macros rely on the fact that directly unreachable

```

Comment:
I recommend "MDEPKG_NDEBUG enabled" instead of "NULL".

Reply:
I find it a bit confusing to talk about enabling something which disables something, I also noted that everywhere else in the file it talks about MDEPKG_NDEBUG being defined or not defined. So I reworded to talk about "The implementations of the macros used when MDEPKG_NDEBUG is defined...". Does this current pushed version now sound okay?